engine/collection/synchronization/engine.go

```diff
@@ -342,7 +345,7 @@ func (e *Engine) processIncomingBlock(originID flow.Identifier, block *clustermo
 		Block:    block,
 	}
 
-	e.comp.SubmitLocal(synced)
+	e.comp.SubmitLocal(engine.ChannelConsensusCluster(e.chainID), synced)
```

Comment:
@jordanschalm

Reply:
Generally, the `SubmitLocal` is for a message transfer from engine to engine, which is _not_ restricted to networking messages. They can be specific node-internal messages. There is no concept for "networking channel" for internal messages.